backend/Wheelingful.Data/DependencyInjection.cs

```diff
@@ -14,4 +15,14 @@ public static void AddDbContext(this IServiceCollection services, IConfiguration
         services.AddDbContext<WheelingfulDbContext>(options =>
             options.UseMySql(config.GetConnectionString("DefaultConnection"), serverVersion));
     }
+
+    public static void AddIdentityDataStores(this IdentityBuilder builder)
+    {
+        builder.AddEntityFrameworkStores<WheelingfulDbContext>();
+    }
+
+    public static void AddDataOuter(this IServiceCollection services)
```

Comment:
do you need this method?

Reply:
It can be deleted, but it will appear again in the near future.